.drone/drone.yml

```diff
@@ -122,6 +122,32 @@ steps:
 depends_on:
   - Test
 
+volumes:
+  - name: docker
+    host:
+      path: /var/run/docker.sock
+---
+kind: pipeline
+type: docker
+name: Deploy-To-Deployment-Tools 
+platform:
+  os: linux
+  arch: amd64
+trigger:
+  ref:
+    - refs/heads/main
+
+steps:
+  - name: Update Deployment Tools 
+    image: us.gcr.io/kubernetes-dev/drone/plugins/updater
+    settings:
```

Comment:
For the curious, these are [converted](https://docs.drone.io/plugins/overview/#plugin-inputs) into `PLUGIN_<settings key>` environment variables for the container to use.